### PR TITLE
Quick (and dirty) copyright fixer

### PR DESCRIPTION
**Pull request by reinkrul**
Got tired of going through the files by hand. It adds the notice to Go files when it's missing.

Run it using:
```go run ./doc copyright```

README generation must now be invoked with:
```go run ./doc docs```

Will run and commit when this is merged.

### CODE REVIEW

File: docs/check_copyright.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+import (
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+const copyrightText = `/*
+ * Copyright (C) 2021 Nuts community
```
Comment (woutslakhorst) [inline docs/check_copyright.go:11]: Use current year and do find/replace?

**Comment by dmeijboom** [comment]
A simple solution using command line utilities would be something like this (not tested):

```
export COPYRIGHT="Copyright contents (newlines should be replaced with \n)"
grep -Ri --include "*.go" -L "Copyright (C) 2021 Nuts community" . | xargs -I{} sed -i "1s;^;${COPYRIGHT}\n;" {}
```

File: docs/check_copyright.go (hunk 1 above)
Comment (reinkrul) [reply docs/check_copyright.go]: Fixed

**Comment by reinkrul** [comment]
> A simple solution using command line utilities would be something like this (not tested):
> 
> ```
> export COPYRIGHT="Copyright contents (newlines should be replaced with \n)"
> grep -Ri --include "*.go" -L "Copyright (C) 2021 Nuts community" . | xargs -I{} sed -i "1s;^;${COPYRIGHT}\n;" {}
> ```

My Bash-fu has a terrible track record, so writing something quickly in Go is much faster for me.

File: docs/check_copyright.go [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+			if info.IsDir() {
+				return nil
+			}
+			if !strings.HasSuffix(info.Name(), ".go") {
+				// only Go files
+				return nil
+			}
+			if strings.Contains(info.Name(), "mock") {
+				return nil
+			}
```
Comment (dmeijboom) [inline docs/check_copyright.go:59]: Nitpick: could be an `or` instead

**Review by dmeijboom — APPROVED** [review]
LGTM if it works

**Pull request merged**